src/howard/howard/issuers/realisation.py

```diff
@@ -1,143 +1,163 @@
 """Realisation Certificate"""
 
+import datetime
 from enum import Enum
+from typing import Generic, TypeVar
+from uuid import UUID
 
 from django.template import Context
 
+import arrow
 from dateutil import parser
+from pydantic import BaseModel, ValidationError
 
 from marion.certificates.issuers.base import AbstractCertificate
 
+ArrowSupportedDateType = TypeVar("ArrowSupportedDateType", int, str, datetime.date)
 
-class CertificateScope(Enum):
-    """Allowed scopes for the RealisationCertificate"""
+
+class Gender(str, Enum):
+    """Gender definitions."""
+
+    MALE = "Mr"
+    FEMALE = "Mme"
+
+    def __str__(self):
+        return f"{self.value}"
```

Comment:
What about factoring a bit?

```suggestion
class StrEnum(str, Enum):
    """String Enum."""

    def __str__(self):
        return f"{self.value}"


class Gender(StrEnum):
    """Gender definitions."""

    MALE = "Mr"
    FEMALE = "Mme"
```

Reply:
Smart ;)